ios/brave-ios/Sources/Brave/Frontend/Rewards/BraveRewards.swift

```diff
@@ -200,6 +200,10 @@ public class BraveRewards: PreferencesObserver {
     }
   }
 
+  public var shouldShowSponsoredImagesAndVideos: Bool {
+    return ads.shouldShowSponsoredImagesAndVideos()
+  }
```

Comment:
This isn't really necessary to wrap, we can just reach into the ads object directly